spec/factories.rb

```diff
@@ -277,11 +289,31 @@
     # belongs_to associations
     association :creator, factory: :lecturer
 
+    # has_many associations
+    association :auto_grading_exact_options, factory: :auto_grading_exact_option
+
     # attributes
     title "Factory-Generated General Question"
     description "Description for factory-generated General question."
     max_grade 1
     staff_comments "Staff comment for factory-generated General question."
+
+    trait :auto_grading_exact do
+      ignore do
+        option_count 2
+      end
+      after(:build) do |question, evaluator|
+        evaluator.option_count.times do
+          question.auto_grading_exact_options << build(:auto_grading_exact_option)
+        end
+      end
+    end
+  end
+
+  factory :auto_grading_exact_option, class: Assessment::AutoGradingExactOption do
+    correct true
+    answer 'Some answer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.